qutip/wigner.py

```diff
@@ -48,12 +48,149 @@
 from qutip.states import ket2dm
 from qutip.parallel import parfor
 from qutip.utilities import clebsch
+from qutip.operators import jmat
 from scipy.special import factorial
 from qutip.cy.sparse_utils import _csr_get_diag
+import qutip as qt
 
 
-def wigner(psi, xvec, yvec, method='clenshaw', g=sqrt(2), 
-            sparse=False, parfor=False):
+def wigner_transformation(psi, j, fullparity, steps, slicearray):
+    """Calculates the wigner function of a quantum state at each theta, phi.
+
+    Parameters
+    ----------
+        psi : qobj
+              a state vector or density matrix.
+        j : int
+            the total angular momentum of the quantum state.
+        sun : int
```

Comment:
What is `sun`?
Also adding `wigner_transformation` to all could be nice.

Reply:
Here `wigner_transform` is defined with 5 parameters but 6 are listed in the description.
`sun` is hard coded to be 2 and not an input, so listing it as a parameter will only cause confusion.

Reply:
Removed this from the docstring in the latest commit.